### PR TITLE
add new projects

### PR DESCRIPTION
**Pull request by rickykuo666**
Please see the [asana ticket](https://app.asana.com/0/1205770721173535/1207664282658139/f)

### CODE REVIEW

File: app/ecosystem/fixture.ts [hunk 1]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+      'decentralized confidential computing (DeCC), enabling dApps to make use of encrypted data.',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/SecretNetwork',
+    website: 'https://scrt.network/'
+  },
+  {
+    image: '/img/ecosystem/Layer3.jpg',
+    title: 'Layer3',
+    description:
+      'Discover the world of crypto like never before. With interactive Quests that make learning and exploring crypto fun, engaging, and rewarding.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:203]:
```suggestion
      'Discover the world of crypto like never before with interactive quests that make learning and exploring crypto fun, engaging, and rewarding.',
```

File: app/ecosystem/fixture.ts [hunk 2]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+      'Pluggable wallet infrastructure for Web3 wallets and applications. Streamline the onboarding of both mainstream and crypto native users in under a minute. ',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/web3auth',
+    website: 'https://web3auth.io/'
+  },
+  {
+    image: '/img/ecosystem/Secret-Network.jpg',
+    title: 'Secret Network',
+    description:
+      'decentralized confidential computing (DeCC), enabling dApps to make use of encrypted data.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:194]:
```suggestion
      'Decentralized confidential computing (DeCC), enabling dApps to make use of encrypted data.',
```

File: app/ecosystem/fixture.ts [hunk 3]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+      'A developer integration toolkit to let users buy/sell crypto in any app, website or web plugin.',
+    categories: ['dev-tools', 'infra'],
+    twitter: 'https://x.com/transak',
+    website: 'https://transak.com/'
+  },
+  {
+    image: '/img/ecosystem/zeeve.png',
+    title: 'Zeeve',
+    description:
+      'An enterprise-grade automation platform providing deployment, monitoring and management of Blockchain nodes and networks.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:176]:
```suggestion
      'An enterprise-grade automation platform providing deployment, monitoring and management of blockchain nodes and networks.',
```

File: app/ecosystem/fixture.ts [hunk 4]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+      'Discover the world of crypto like never before. With interactive Quests that make learning and exploring crypto fun, engaging, and rewarding.',
+    categories: ['social'],
+    twitter: 'https://x.com/layer3xyz',
+    website: 'https://layer3.xyz/'
+  },
+  {
+    image: '/img/ecosystem/MintyCodeDAO.jpg',
+    title: 'Mintycode.io',
+    description:
+      'Decentralised Marketplace for Open-Source Creators, Users, and Contributors.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:212]:
```suggestion
      'Decentralised marketplace for open-source creators, users, and contributors.',
```

File: app/ecosystem/fixture.ts [hunk 5]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+    categories: ['social'],
+    twitter: 'https://x.com/MintyCodeDao',
+    website: 'http://mintycode.io/'
+  },
+  {
+    image: '/img/ecosystem/UHT.jpg',
+    title: 'UHT',
+    description:
+      'Gamify health, reward healthy behaviour & lifestyle habits with UHT (Universal Health Token).',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:221]:
```suggestion
      'Gamify health and reward healthy behaviour & lifestyle habits with UHT (Universal Health Token).',
```

File: app/ecosystem/fixture.ts [hunk 6]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+    categories: ['social'],
+    twitter: 'https://x.com/UHTofficial',
+    website: 'https://universalhealthtoken.xyz/'
+  },
+  {
+    image: '/img/ecosystem/Open Ticketing Protocol Logo.jpg',
+    title: 'Open Ticketing Protocol (GET Protocol)',
+    description:
+      'The home of onchain ticketing. Bringing accessible onchain benefits to millions of event attendees, organizers and artists around the world.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:230]:
```suggestion
      'The home of onchain ticketing, bringing accessible onchain benefits to millions of event attendees, organizers and artists around the world.',
```

File: app/ecosystem/fixture.ts [hunk 7]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+    title: 'TachySwap',
+    description: 'AMM based decentralized exchange.',
+    categories: ['defi'],
+    twitter: 'https://x.com/tachyswap',
+    website: 'https://tachyswap.org/'
+  },
+  {
+    image: '/img/ecosystem/cobweb-labs.jpg',
+    title: 'CobWeb Labs',
+    description: 've(3,3) DEX.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:246]: Can you do any better with the description here?

File: app/ecosystem/fixture.ts [hunk 8]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+      'Full-stack Web3 infrastructure – node infra, exploration tools & dev environments.',
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/TenderlyApp',
+    website: 'https://tenderly.co/'
+  },
+  {
+    image: '/img/ecosystem/Bithotel.jpg',
+    title: 'Bithotel',
+    description:
+      'Revolutionary social pixelart game (free-to-play!) on Mobile and PC.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:264]:
```suggestion
      'Revolutionary social pixelart game (free-to-play!) on mobile and PC.',
```

File: app/ecosystem/fixture.ts [hunk 9]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+    description:
+      'A web3 protocol for organizing and accessing blockchain data.',
+    categories: ['infra'],
+    twitter: 'https://x.com/graphprotocol',
+    website: 'https://thegraph.com'
+  },
+  {
+    image: '/img/ecosystem/Omnisea.jpg',
+    title: 'Omnisea',
+    description: 'Omnichain NFT Launchpad powered by LayerZero.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:299]:
```suggestion
    description: 'Omnichain NFT launchpad powered by LayerZero.',
```

File: app/ecosystem/fixture.ts [hunk 10]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+    categories: ['nfts'],
+    twitter: 'https://x.com/omnisea',
+    website: 'https://omnisea.org/'
+  },
+  {
+    image: '/img/ecosystem/groupfi.jpg',
+    title: 'GroupFi',
+    description:
+      'A Web3 messaging protocol designed by the Tanglepay Team for seamless, feeless, on-chain and cross-application group communication. ',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:308]: In some places you use "on-chain" and in others you use "onchain" -- be consistent.

File: app/ecosystem/fixture.ts [hunk 11]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+    categories: ['dev-tools'],
+    twitter: 'https://x.com/groupfi_ai',
+    website: 'https://groupfi.ai/'
+  },
+  {
+    image: '/img/ecosystem/ChessPlay.jpg',
+    title: 'Chess Play',
+    description:
+      'Chess meets blockchain. Your destination for innovative gameplay and secure transactions. Reshaping the future of on chain chess!',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:317]:
```suggestion
      'Chess meets blockchain: your destination for innovative gameplay and secure transactions, reshaping the future of onchain chess!',
```

Just to make these complete sentences.

File: app/ecosystem/fixture.ts [hunk 12]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+    categories: ['gaming'],
+    twitter: 'https://x.com/play_chess_/',
+    website: 'https://www.chessplay.live/'
+  },
+  {
+    image: '/img/ecosystem/Gamefi.jpg',
+    title: 'GameFi',
+    description: 'A One-stop Destination Launchpad For Web3 Enthusiasts.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:325]:
```suggestion
    description: 'A one-stop destination launchpad for Web3 enthusiasts.',
```

File: app/ecosystem/fixture.ts [hunk 13]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+    categories: ['launchpad'],
+    twitter: 'https://x.com/gamefi_official?lang=en',
+    website: 'https://gamefi.org/'
+  },
+  {
+    image: '/img/ecosystem/Openpad.jpg',
+    title: 'OpenPad',
+    description:
+      'Decentralized Crowdfunding Ecosystem + Onestop Incubation Hub for Web3 Projects.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:334]:
```suggestion
      'Decentralized crowdfunding ecosystem + on-stop incubation hub for Web3 projects.',
```

"Web3" is capitalized here but not elsewhere.

File: app/ecosystem/fixture.ts [hunk 14]
```diff
@@ -150,5 +150,217 @@ export const projects: ProjectCardProps[] = [
...
+      'A personalized block viewing tool for inspecting and analyzing EVM and non-EVM blockchain data & transactions.',
+    categories: ['infra'],
+    twitter: 'https://x.com/tracehawk_io',
+    website: 'https://www.tracehawk.io/'
+  },
+  {
+    image: '/img/ecosystem/TRACEEYE.png',
+    title: 'Traceye',
+    description:
+      'An enterprise-grade Blockchain Infrastructure Automation Platform.',
```
Comment (timothymcmackin) [inline app/ecosystem/fixture.ts:361]:
```suggestion
      'An enterprise-grade blockchain infrastructure automation platform.',
```

**Review by timothymcmackin — CHANGES_REQUESTED** [review]

File: app/ecosystem/fixture.ts (hunk 7 above)
Comment (rickykuo666) [reply app/ecosystem/fixture.ts:246]: Let me double check with Aaron

File: app/ecosystem/fixture.ts (hunk 13 above)
Comment (rickykuo666) [reply app/ecosystem/fixture.ts:334]: changed all to Web3

**Review by timothymcmackin — APPROVED** [review]

**Pull request merged**